app/services/alerts/adapters/report.rb

```diff
@@ -0,0 +1,17 @@
+module Alerts
+  module Adapters
+    class Report
```

Comment:
There has got to be a simpler way of defining classed like this but I'm not sure of what direction to take. Any preferences?

Reply:
You mean without the module module?

Reply:
No, I prefer module module over the rails autoloady magic way of doing modules.

I was thinking of an easier way to define boilerplate PORO-ish classes. Something activemodel based?

Reply:
Ah yes, maybe, I always forget about what you get with ActiveModel when you use it without AR

Reply:
Unfortunately, you don't get value defaults with ActiveModel